src/Processors/IMSCP/Entities/Manifest.php

```diff
@@ -9,6 +9,23 @@ class Manifest
     protected $metadata;
     protected $organizations;
     protected $manifest;
+    protected $imsManifestMetadata;
```

Comment:
Could you add some docblock description for both this and `$metadata` (and possibly `$manifest` as well) to explain the difference between them for future readers?

Reply:
Done